agent/engine/docker_task_engine.go

```diff
@@ -945,6 +961,22 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 	return metadata
 }
 
+func getLogRouterConfig(task *apitask.Task, container *apicontainer.Container, hostConfig *dockercontainer.HostConfig, engine *DockerTaskEngine) dockercontainer.LogConfig {
```

Comment:
can you add a unit test for this method? 
nit: "engine *DockerTaskEngine" argument can just be "cfg *config.Config" since we only use the config rather than other part of the engine.

Reply:
Talked about this offline.